src/lib/Components/Bidding/__stories__/BidFlow.story.tsx

```diff
@@ -56,24 +56,28 @@ const ConfirmBidScreenRenderer: React.SFC<any> = ({ render, saleArtworkID }) =>
 
 storiesOf("Bidding")
   .add("Show bid flow", () => {
-    return <BidFlowRenderer render={renderWithLoadProgress(BidFlow)} saleArtworkID="5aada729139b216c0bf18103" />
+    return <BidFlowRenderer render={renderWithLoadProgress(BidFlow)} saleArtworkID="5ae72c86ae68a93562b5706f" />
   })
   .add("Select Max Bid", () => (
-    <SelectMaxBidRenderer render={renderWithLoadProgress(MaxBidScreen)} saleArtworkID="5aada729139b216c0bf18103" />
+    <SelectMaxBidRenderer render={renderWithLoadProgress(MaxBidScreen)} saleArtworkID="5ae72c86ae68a93562b5706f" />
   ))
   .add("Confirm Bid", () => {
     return (
       <ConfirmBidScreenRenderer
         render={renderWithLoadProgress(ConfirmBidScreen, { bid: { display: "$20,000", cents: 2000000 } })}
-        saleArtworkID="5aada729139b216c0bf18103"
+        saleArtworkID="5ae72c86ae68a93562b5706f"
```

Comment:
#minor (and not related to this PR per se), but since we re-set staging each week, this ID is going to become outdated 😞 (I'm assuming that this pulls real staging data into the storybooks, right?)